committee-steering/governance/annual-reports.md

```diff
@@ -26,31 +26,41 @@ ensure that reports are complete, accurate, and submitted to the Steering
 Committee.
 
 ### 2020:
-Due to playing catch up with 40 community groups, we will have a slightly
+Due to playing catch up with 37 community groups, we will have a slightly
 modified timeline bootstrap process this year and then will follow the process
-outlined in 2021. This means that Chairs will be reporting twice in n months and
-not a year. A yearly cadence will be followed post 2021.
-
-TODO outline time line for 2020  
-
-### 2021 and on:
-1. Chairs create one report and send it to steering-private@kubernetes.io for
-review; reports are due at the end of February and will use a template with the
-questions listed below TODO: to be created
-2. Steering Committee will assign one member as a liaison to each community group before the
-reports are submitted so there is an established point of contact for any help or
-guidance needed
-3. The March SIG Chairs and Tech Leads meetings will be used as follow up for
-the community groups that have questions from/to Steering and report
-retrospectives
-4. Steering will schedule private meetings to follow up on anything denoted as
+outlined for 2021. Working Groups, due to their temporary nature, will be the only group 
+to submit for 2020.   
+
+1. Steering Committee will assign one member as a liaison to each community group for kick 
```

Comment:
Not for this PR, but commenting here to ensure we don't forget -- we should PR in the list of liaisons in the steering repo before July 1st (https://github.com/kubernetes/steering/issues/159).

Reply:
Created https://github.com/kubernetes/steering/pull/168